### DISCUSSION TITLE
how to query by multi relation field?

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2247] [Q&A]
`typeIds` is multi relation field. I wanna select records `typeIds` contain  ID `0`.

`database.App.Dao().FindRecordsByExpr("users", dbx.Like("typeIds", "0"))`

**Comment by ganigeorgiev** [comment]
Multiple `reltion` fields are stored as serialized json array and if you want to apply a condition on each array item you'll have to perform a join with each element via [`json_each`](https://www.sqlite.org/json1.html#jeach).

You can find a code example in https://github.com/pocketbase/pocketbase/discussions/1959#discussioncomment-5182090.
